lib/resources/pix.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * @name Pix
+ * @description This module exposes functions
+ *              related to the `/pix` path.
+ *
+ * @module balanceOperations
+ **/
+import routes from '../routes'
+import request from '../request'
+
+const createKey = (opts, body) =>
```

Comment:
```suggestion
const create = (opts, body) =>
```

sugestão de alterar para manter a consistencia com outros recursos já mapeados